bank_marketing/decisionTree/train.py

```diff
@@ -6,30 +6,43 @@
 from bank_marketing.common_utils.train_utils import Encoder
 from utils.encode_decode import pickle_model
 
+
 def train(msg):
-    random.seed(0)
```

Comment:
I would be inclined to leave random seeding in even though we're not explicitly using it, because some models may be stochastic in their training and hence use it implicitly, so putting it in at worst does not harm and in some cases will be necessary to ensure deterministic training results

Reply:
@sdraper-CS ah that makes sense. Also this `random.seed` comes from `random` module and not `np.random`. Should we be using `np.random` or vanilla `random` package ?

Reply:
Set both random seeds is usually the best advice (and pytorch's too if you also use that, though we don't here - same goes for TF too I would think)